remix-lib/src/universalDapp.js

```diff
@@ -271,14 +272,11 @@ module.exports = class UniversalDApp {
         }
         this.silentRunTx(tx, (error, result) => {
           if (error) return reject(error)
-          resolve({
-            transactionHash: result.transactionHash,
-            status: result.result.status,
-            gasUsed: '0x' + result.result.gasUsed.toString('hex'),
-            error: result.result.vm.exceptionError,
-            return: result.result.vm.return ? '0x' + result.result.vm.return.toString('hex') : '0x',
```

Comment:
Although previously, it was returning `0x` in case of non-existing value and now it will return `undefined`, we can go with that.

Reply:
`0x` makes sense. Should I make it return `0x` for all the missing values? (except maybe error, that could stay undefined)

Reply:
No, we can go with the `undefined`, that's fine.